stylesheets/global-style.css

```diff
@@ -0,0 +1,374 @@
+  list-style-type: none;
+}
+
+nav ul li {
+	float: left;
+	color: #FFFFFF;
+	margin-right: 2%;
+	letter-spacing: 2px;
+}
+ul li a {
+  text-decoration: none;
+}
+
+.dog-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  float: left;
+  grid-gap: 30px;
+}
+
+.dog-box button {
+  background-color: #99CC33;
+  color: white;
+  font-style: oblique;
+  font-weight: bold;
+  text-align: center;
+}
+
+.dog-box {
+  padding: 0px;
+  border: 1px solid gray;
+  margin: 5%;
+  float: left;
+}
+
+.dog-box p {
+  padding-left: 10px;
+  text-align: left;
+}
+
+.dog-box h3 {
+  padding-left: 10px;
+  text-align: left;
+  margin-left: 2px;
+  color: #2952a3;
+}
+
+.dog-box h4 {
+  padding-left: 10px;
+  text-align: left;
+}
+
+.dog-box button {
+  display: inline-block;
+  width: 95%;
+  margin: 5px;
+}
+
+.hero {
+  width: 100%;
+  padding: 10px;
+  box-sizing: border-box;
+  overflow:hidden;
+  text-align: left;
+}
+
+.aside {
+  float: left;
+  width: 55%;
+}
+
+.hero-text {
+  float: left;
+  margin-left: 10px;
+}
+
+.blog-post {
+  width: 100%;
+  overflow:hidden;
+  text-align: left;
+}
+
+.image-section,
+.aside {
+  margin: 10px;
+}
+
+aside h2 {
+  color: #2952a3;
+}
+
+.image-section {
+  float: left;
+  width: 30%;
+  margin-left: 5%;
+  padding: 10px;
+}
+
+aside {
+  float: right;
+  width: 25%;
+  min-width: 10%;
+  padding: 20px;
+}
+
+.dogs-adopted {
+  width: 10%;
+  padding: 0px;
+  border: 1px solid gray;
+  box-sizing: border-box;
+  margin: auto;
+}
+
+form {
+  color: #2952a3;
+  font-weight: bold;
+}
+
+.text-input {
+  width: 50%;
+  height: 25px;
+}
+
+.checkout-dogs {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.footer {
+  background-color: #383838;
+  align-items: center;
+  color: white;
+  width: 100%;
+  clear: both;
+}
+
+.footer img {
+  align-items: center;
+  width: 30%;
+  padding-top: 3%;
+}
+
+.footer a {
+  font-size: 25px;
+  color: white;
+  font-variant: small-caps;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.footer p {
+  padding-bottom: 3%;
+}
+
+h1 {
+  color: #2952a3;
+}
+
+main {
+  max-width: 70%;
+  float: left;
+}
+
+/* for-phone-only */
+@media (max-width: 599px) {
+
+  .dog-container,
+  .dog-box {
+    grid-template-columns: 100%;
+    grid-template-rows: repeat(1, auto);
+  }
+
+  aside {
+    display: none;
+  }
+}
+
+/* for-tablet-landscape { */
+@media (min-width: 900px) {
+  .dog-container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .dog-box {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(1, auto);
+  }
+
+  .aside {
+    float: left;
+    width: 55%;
+  }
+
+  .aside img {
+    width: 25%;
+  }
+}
+
+/* for-desktop { */
+@media (min-width: 1200px) {
+  .dog-container {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .dog-box {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+/* for-big-desktop-up { */
+@media (min-width: 1800px) {
+  .dog-box {
+    grid-template-columns: repeat(6, 1fr);
```

Comment:
Same here, applying grid layouts to .dog-box has no effect, target .dog-container

Reply:
Dog images are not filling out cards well at larger resolutions, but that should be fixed once you change the .dog-container to adjust to 6 columns.